src/app/app.component.spec.ts

```diff
@@ -0,0 +1,24 @@
+import { TestBed } from '@angular/core/testing';
+import { AppComponent } from './app.component';
+
+describe('AppComponent', () => {
+  beforeEach(async () => {
+    await TestBed.configureTestingModule({
+      declarations: [
+        AppComponent
+      ],
+    }).compileComponents();
+  });
+
+  it('should create the app', () => {
```

Comment:
Does this test make sense? We don't need to test a specific framework like Angular. I think we need to test our own logic. In this patch, we don't have any logic yet.

Reply:
it generated aytomatically by angular f/w will remove it